### DISCUSSION TITLE
Minimum Hardware/Resource recommendations?

### DISCUSSION THREAD

**Discussion by danthegoodman1** [ory/kratos#1745] [Q&A] [answered]
In the [Multitenancy](https://www.ory.sh/kratos/docs/guides/multi-tenancy-multitenant/) docs it is mentioned that Kratos is very lightweight. I was curious how light weight?

I know performance metrics have intentionally not been added, and I am sure that a lot comes down the the DB performance backing it, however I am wondering (in the context of k8s specifically) how many resources I need to allocate?

For example, if I wanted a really minimal deploment, would `100m` CPU and `256MB` RAM suffice? What would suggested minimum requirements be?

If I wanted to hit ~100 checks/s, how much CPU and RAM might I want (ignoring the DB requirements)?

**Comment by aeneasr** [comment, marked as answer]
The most expensive operation is password hashing. Depending on how many hashes you want to do per second, and what algorithm you use with which setting, the outcome would be different. Without hashing, the params given would absolutely suffice!

**Reply by danthegoodman1** [reply]
Hashing is only done on register and login, correct?

**Reply by aeneasr** [reply]
Exactly!
